src/client/tweetAnalysis.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+/**
+ * Given some list of tweets, create a dictionary of how often each word appears
+ * @param tweets An array of Tweet objects
+ * @returns {{}} e.g. {'hello': 5, 'world': 8}
+ */
+function countWords(tweets) {
+  const wordCount = {};
+  for (let tweet of tweets) {
+    for (let word of tweet.content.split(/[ !,.?]/)) {

```

Comment:
@VasilyShelkov can you think of other punctuation we should ignore? What should we do about things "in speech marks"?